apps/coscrad-frontend-e2e/src/e2e/resources/photographs/queries/photograph.detail.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,151 @@
+        });
+
+        it(`should display the not found page`, () => {
+            cy.getByDataAttribute('not-found');
+        });
+    });
+
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(200);
+
+        cy.seedDataWithCommand(`CREATE_MEDIA_ITEM`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifier,
+            title: 'My photo of Masset',
+            mimeType: MIMEType.jpg,
+            // Override default value to "remove" property from fsa for image media item
+            lengthMilliseconds: null,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifier,
+        });
+
+        cy.seedDataWithCommand(`CREATE_PHOTOGRAPH`, {
+            aggregateCompositeIdentifier,
+            title: photographTitle,
+            languageCodeForTitle,
+            mediaItemId: mediaItemCompositeIdentifier.id,
+            heightPx: 800,
+            widthPx: 200,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier,
+        });
+
+        cy.seedDataWithCommand(`CREATE_NOTE_ABOUT_RESOURCE`, {
+            aggregateCompositeIdentifier: buildDummyAggregateCompositeIdentifier(
+                AggregateType.note,
+                15
+            ),
+            resourceCompositeIdentifier: aggregateCompositeIdentifier,
+            text: noteText,
+        });
+
+        cy.seedDataWithCommand(`CREATE_MEDIA_ITEM`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifierNoNotes,
+            title: 'My photo of Skidegate with no notes',
+            mimeType: MIMEType.jpg,
+            // Override default value to "remove" property from fsa for image media item
+            lengthMilliseconds: null,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: mediaItemCompositeIdentifierNoNotes,
+        });
+
+        cy.seedDataWithCommand(`CREATE_PHOTOGRAPH`, {
+            aggregateCompositeIdentifier: aggregateCompositeIdentifierNoNotes,
+            title: photographTitleNoNotes,
+            languageCodeForTitle,
+            mediaItemId: mediaItemCompositeIdentifierNoNotes.id,
+            heightPx: 800,
+            widthPx: 200,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: aggregateCompositeIdentifierNoNotes,
+        });
+    });
+
+    describe(`when the photograph exists`, () => {
+        beforeEach(() => {
+            cy.visit(validPhotographDetailRoute);
+        });
+
+        it(`should contain the photograph name`, () => {
+            cy.contains(photographTitle);
+
+            cy.getByDataAttribute('LanguageIcon').first().trigger('mouseover');
+
+            cy.contains("Haida, 'original'");
+        });
+    });
+
+    describe('when there are notes for the Photograph (2)', () => {
```

Comment:
What about connections?

Reply:
Yeah, I'll add that in.  I wasn't sure what to make as part of the scope here.